ip_access_rules.go

```diff
@@ -0,0 +1,97 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/goccy/go-json"
+)
+
+type ListIpAccessRulesOrderOption string
+type ListIpAccessRulesMatchOption string
+type RulesModeOption string
+
+const (
+	ConfigurationTarget      ListIpAccessRulesOrderOption = "configuration.target"
+	ConfigurationValue       ListIpAccessRulesOrderOption = "configuration.value"
```

Comment:
these aren't strings; it is nested object notation.

Reply:
Hmmm. That's not what the API docs say. Maybe they are wrong:


![Screenshot 2023-10-27 at 09 07 02](https://github.com/cloudflare/cloudflare-go/assets/6424250/0c14f908-48ba-4da0-8d12-8a9506b1e672)

https://developers.cloudflare.com/api/operations/ip-access-rules-for-a-zone-list-ip-access-rules

Reply:
the documentation is using incorrect notation. if you look at the examples (and the live API calls), it is a nested JSON block.

Reply:
@jacobbednarz sorry for slow response, didn't see email from github. I really don't see how that's possible. This is a `string` in which you are specifying how to order the results. Nested JSON is part of the IP access rule itself but not the `?order=X` param, unless I'm really missing something here.

Reply:
ah yes, we're looking at two different things here.